app/presenters/curation_concerns/inspect_work_presenter.rb

```diff
@@ -0,0 +1,64 @@
+module CurationConcerns
+  class InspectWorkPresenter
+    def initialize(solr_document, current_ability)
+      @solr_document = solr_document
+      @current_ability = current_ability
+    end
+    attr_reader :solr_document, :current_ability
+
+    def workflow
+      if sipity_entity
+        {
+          entity_id: sipity_entity.id,
+          proxy_for: sipity_entity.proxy_for_global_id,
+          workflow_id: sipity_entity.workflow_id,
+          workflow_name: sipity_entity.workflow_name,
+          state_id: sipity_entity.workflow_state_id,
+          state_name: sipity_entity.workflow_state_name,
+          comments: workflow_comments,
+          roles: sipity_role # TODO: How to get related roles and actors for the sipity_entity
+        }
+      end
+    end
+
+    def solr
+      work.to_solr if work
```

Comment:
I'm not sure we want to tackle that right now.  We should emphasize the workflow.